doc/api/process.md

```diff
@@ -1982,6 +1982,19 @@ A boolean value that is `true` if the current Node.js build includes support for
 
 ***
 
+## `process.features.typescript`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean|string}
+
+A string / boolean value that is `"strip"` if Node.js is run with `--experimental-strip-types`,
+`"transform"` if Node.js is run with `--experimental-transform-types`, and `false` otherwise.
```

Comment:
Can we make it `undefined` so if we have to remove it, it won't break anyone?
```suggestion
`"transform"` if Node.js is run with `--experimental-transform-types`, and `undefined` otherwise.
```

Reply:
See https://github.com/nodejs/node/pull/54295#discussion_r1724751168

Reply:
@legendecas wdyt?

Reply:
Its experimental so its fine to have it return `false`

Reply:
Currently it's not documented as experimental

Reply:
I don't think the process.features part should emit a warning. I'm okay documenting it as experimental?

Reply:
Yes, thats what I meant, sorry about the wording

Reply:
I am not in favor of a property presence but with its value be `undefined` intentionally. I don't find it improving breakage reduction since... this property is still enumerable and can be detected with `in` operator. IMO `false` is explicit about disabled.